source/10-compliance/03-counter-reporting-for-consortia.rst

```diff
@@ -7,38 +7,52 @@
+* Report providers may elect to offer detailed reports that show total usage for all members of the consortium rolled up to the consortia level, and which can be broken down by institution. As with the summary report, the totals on the detailed report may differ from the sum of the totals on individual member reports for the same items.
+* Institutional reports are the individual institutional-level reports. Consortia managers and administrators MUST be able to retrieve institutional reports.
 
-Privacy and Confidentiality
-"""""""""""""""""""""""""""
+Note that consortia-level reports are restricted to COUNTER Reports. The Standard Views of the COUNTER Reports are not available as consortia-level reports.
 
-COUNTER acknowledges that some organizations treat their usage data as sensitive and private information. Content providers may include the option for consortium members to opt-out of consortium reporting. COUNTER recommends the default setting for an organization is to opt-in to consortium reporting.
 
+Creating Detailed Consortia-Level Reports
+"""""""""""""""""""""""""""""""""""""""""
 
-Content to Report Usage On
-""""""""""""""""""""""""""
+COUNTER recognizes that consolidated reports can be helpful to consortia, and that some report providers may want to meet the needs of their consortium customers by providing detailed reports. This can be done using existing extensions to the Code of Practice (see :numref:`reserved-elements`):
 
-When a COUNTER report is harvested by a consortium administrator, a content provider may choose to limit member usage to include only content acquired through the consortium. Note that when such a limitation is in place the resulting report may differ from the member-sites own version of the report. Since not all content providers can provide such limits, the consortium will be responsible for ensuring usage is filtered to the content they license for members.
+* The Customer_ID used in a detailed report MUST match that used in a report to the individual institution, and the ID for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, the Institution_Name used in a detailed report should match that used in a report to the individual institution, and the name for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, Institution_Name should be the first column in the tabular report, with Customer_ID the second column. Otherwise Customer_ID should be the first column. 
 
-When the content provider chooses to limit member usage to only content acquired through the consortium, they MUST include a message to this effect in the Notes element in their implementation of the /members path in the COUNTER_SUSHI API (see :numref:`sushi` above).
```

Comment:
In my opinion this is an important role that we should keep. We could either keep it in this section or move it to the /members description in Section 8.1.

Reply:
The line wasn't deleted, just moved to line 56 under the heading content to report usage on